character.cpp

```diff
@@ -1,6 +1,6 @@
 #include "character.h"
 
-Character::Character(std::string name, int health, unsigned int damage) :
+Character::Character(const std::string name, int health, const unsigned int damage) :
```

Comment:
itt annyira amugy nem fontosak a const-ok, sot, kicsi furak is, de nem egetvero baj.